src/components/Footer/Footer.js

```diff
@@ -13,7 +13,7 @@ function Footer(){
 					</div>
 					<div className="footer-item">
 						<h3>Reach us at</h3>
-						<a href="mailto: acm@ucla.edu" className="email"><span>acm@ucla.edu</span></a>
+						<a href="mailto: acm@ucla.edu" className="email"><span className="footer-text">acm@ucla.edu</span></a>
```

Comment:
Curious what the forcing function is for this change

Reply:
Great q! It's because there's a very generic `span` selector [in this file](https://github.com/uclaacm/website/pull/148/files#diff-965075ff141806e2de29cf289b87c2459a7d1ba18a0fa5257b2ae0beb622c8feR59) that was being applied to the buttons inadvertently, so I specified it.